src/test/java/org/logstash/filters/DateFilterTest.java

```diff
@@ -35,6 +49,40 @@ public void testIsoStrings() throws Exception {
     }
 
     @Test
+    public void testPatternStringsInterpolateTzNoYear() throws Exception {
+        Map<String, String> testElements = new HashMap<String, String>() {{
+            put("Mar 27 01:59:59.999", "2016-03-27T00:59:59.999Z");
+//            put("Mar 27 02:00:01.000", "2016-03-27T01:00:01.000Z"); // this should and does fail, the time does not exist
```

Comment:
I would like to refactor the `applyString...` test method so it can know to assert that a SUCCESS or FAIL is a good outcome. If I create an issue for this can I leave it in?